CreateDB.sql

```diff
@@ -1,32 +1,39 @@
 
-CREATE TABLE feedback(
-    feedback_id varchar(36),
+CREATE TABLE IF NOT EXISTS projects(
     project_id varchar(22),
-    user_id varchar(36),
+    primary key (project_id)
+);
+
+CREATE TABLE IF NOT EXISTS feedbacks(
+    feedback_id uuid,
```

Comment:
I'd add default UUID generation, `feedback_ui uuid NOT NULL DEFAULT gen_random_uuid()` with the extension `pgcrypto` as described [here](https://dba.stackexchange.com/a/223124). Also I'd explicitly add `NOT NULL`, to make it more obvious.